### ISSUE TITLE
Need full example setup civet on sveltekit

### ISSUE THREAD

**Issue by TrungRueta** [DanielXMoore/Civet#1274]
Hi, I'm new to Civet (actually I just found Civet yesterday) and I love it because the code style is concise! But I'm in hard to follow because I have not seen any guidelines on setting up Civet work with Sveltekit.

Has anyone successfully set up or written guidelines on how to do it?

Thank you!

**Comment by edemaine** [comment]
Closing this in favor of #351, but feel free to continue discussion there!
